Support/Testing/Travis/script.sh

```diff
@@ -14,6 +14,17 @@ source "$top/Support/Scripts/common.sh"
 
 cformat="clang-format-4.0"
 
+# We're running out of disk space on Travis. This is a dirty, dirty hack. We shouldn't have to do this.
+# In the home directory, /home/travis/perl5 takes up roughly 1.1GB of space and /home/travis/otp takes
+# up roughly 3.0GB. We should be able to recover this space on travis by removing these things, since
+# we never use them.
+if [ -e "~/perl5" ]; then
```

Comment:
There's no need to check if the folder exists because you're using `rm -f` anyway.